source/mainnet/net/guides/remove-delegation.rst

```diff
@@ -0,0 +1,154 @@
+.. _remove-delegation:
+
+=======================================================
+Remove delegation to a baker pool or passive delegation
+=======================================================
+
+You can remove the delegation on an account to stop delegating.
+
+If you remove the delegation, there is a :ref:`cool-down period<glossary-cool-down-period>` before the delegation is stopped. During this period, you continue to earn rewards; you cannot change the delegation amount, but you can change other delegation settings. The delegation stop is not effective until the next pay day after cool-down period ends. At that time the amount you had delegated is returned to your disposable balance.
+
+.. Note::
+
+   All transfers and transactions cost a fee, including staking and unstaking transactions. The fee is based on the set NRG for that transaction and the current exchange rate.
+   The cost of transaction fees is stable in Euros, and therefore the price in CCD varies depending on the CCD to EUR exchange rate. The fee will always be deducted from the **Balance** of the account, so it is important to have some available CCDs to cover fees. A locked-for-staking balance cannot be used to pay for these transactions.
+   You can see the fee in the transaction log.
+
+.. Warning::
+   Make sure you have enough funds in your disposable balance to cover transaction fees.
```

Comment:
Is this actually relevant for removing delegation, I assume it's to avoid users staking everything, but here they are removing it :shrug:

Reply:
Yeah, but if they have sent all their disposable funds somewhere else they will have a hard time removing delegation.